components/producers/ossf-scorecard/main.go

```diff
@@ -4,6 +4,7 @@ import (
 	"encoding/json"
 	"fmt"
 	"log"
```

Comment:
I see that we're using both `log` and `slog`, can we just use `slog`?

Reply:
there's no slog.Fatal